### PR TITLE
Tools: Review resource strings

### PR DESCRIPTION
**Pull request by bricelam**
Part of #7201

@Pilchie FYI

### CODE REVIEW

**Comment by Pilchie** [comment]
Approved for .NET 5 RC2.

File: src/EFCore.Tools/tools/EntityFrameworkCore.PS2.psm1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ $versionErrorMessage = 'The Entity Framework Core Package Manager Console Tools
     The startup project to use. Defaults to the solution's startup project.
 
 .PARAMETER Namespace
-    Specify to override the namespace for the migration.
+    The namespace to use. Defaults to match the directory.
```
Comment (AndriySvyryd) [inline src/EFCore.Tools/tools/EntityFrameworkCore.PS2.psm1:29]:
```suggestion
    The namespace to use. Matches the directory name by default.
```

File: src/EFCore.Tools/tools/EntityFrameworkCore.PS2.psm1 [hunk 2]
```diff
@@ -243,10 +244,10 @@ function Remove-Migration(
     The startup project to use. Defaults to the solution's startup project.
 
 .PARAMETER Namespace
-    Specify to override the namespace for the generated entity types.
+    The namespace to use. Defaults to match the directory.
 
 .PARAMETER ContextNamespace
-    Specify to override the namespace for the DbContext class.
+    The namespace of the DbContext class. Defaults to match the directory.
```
Comment (AndriySvyryd) [inline src/EFCore.Tools/tools/EntityFrameworkCore.PS2.psm1:250]: Ditto for all of these

File: src/dotnet-ef/Properties/Resources.resx [hunk 3]
```diff
@@ -259,7 +262,7 @@
     <value>Startup project '{startupProject}' targets framework '.NETStandard'. There is no runtime associated with this framework, and projects targeting it cannot be executed directly. To use the Entity Framework Core .NET Command-line Tools with this project, add an executable project targeting .NET Core or .NET Framework that references this project, and set it as the startup project using --startup-project; or, update this project to cross-target .NET Core or .NET Framework. For more information on using the Entity Framework Tools with .NET Standard projects, see https://go.microsoft.com/fwlink/?linkid=2034781</value>
   </data>
   <data name="NoBuildDescription" xml:space="preserve">
-    <value>Don't build the project. Only use this when the build is up-to-date.</value>
+    <value>Don't build the project. Only use when the build is up-to-date.</value>
```
Comment (AndriySvyryd) [inline src/dotnet-ef/Properties/Resources.resx:265]:
```suggestion
    <value>Don't build the project. Intended to be used when the build is up-to-date.</value>
```

**Review by AndriySvyryd — APPROVED** [review]

**Pull request merged**